PolicyProject1/src/main/java/com/example/demo/controller/PolicyController.java

```diff
@@ -1,33 +1,28 @@
 package com.example.demo.controller;
 
+import com.example.demo.model.request.CreatePolicyRequest;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
-
 import com.example.demo.model.Policy;
 import com.example.demo.service.PolicyService;
 
 @RestController
-
-@RequestMapping("task")
-
-public class PolicyController 
+@RequestMapping("/policy")

```

Comment:
URL naming should be done properly

Reply:
PolicyController class should have Controller level path defined and all the REST resources inside it should use the same!